### ISSUE TITLE
cannot compile opentyrian

### ISSUE THREAD

**Issue by as-muncher** [opentyrian/opentyrian#59]
Hi, I tried to compile opentyrian on Microsoft Visual Studio 2022, and got a number of errors, I think, because the system couldn't find, I think it was SDL.h  or some .h file.

**Comment by carlreinke** [comment]
You need to set the path to the SDL and SDL_net development packages in "visualc/sdl_paths.props".

**Comment by as-muncher** [comment]
Also, I'm sorry, but where are these SDL and SDL.net development packages?  I couldn't seem to download them off the web, and they're not included in opentyrian's source code, right?

**Comment by as-muncher** [comment]
Really dumb it down for me please, @carlreinke.  So far, I have 2.1.20130907 which works on my system, but haven't done the Git Checkout to see which version I can at least compile without SDH.  So please tell me how to get the lastest version of opentyrian running on my system.  I am frustrated with Visual Studio 2022 which does not show sdl_paths.props in the Solution Explorer when I search for it.  very frustrating.  I see it there when I search for it using Everything, but Visual Studio 2022 should show it.

**Comment by carlreinke** [comment]
[SDL2-devel-2.26.2-VC.zip](https://github.com/libsdl-org/SDL/releases/download/release-2.26.2/SDL2-devel-2.26.2-VC.zip) and [SDL2_net-devel-2.2.0-VC.zip](https://github.com/libsdl-org/SDL_net/releases/download/release-2.2.0/SDL2_net-devel-2.2.0-VC.zip)
